src/javascript/app/Stores/Modules/Contract/contract-store.js

```diff
@@ -169,10 +173,19 @@ export default class ContractStore extends BaseStore {
             this.contract_info = {};
             this.contract_id   = null;
             this.smart_chart.setContractMode(false);
+            this.smart_chart.setIsChartLoading(false);
             return;
         }
         if (+response.proposal_open_contract.contract_id !== +this.contract_id) return;
 
+        // Clear chart loading status once ChartListener returns ready for completed contract
+        if (this.smart_chart.is_chart_ready) {
+            // TODO: Remove timeout once issue with moving chart line is fixed when loading completed contract
+            setTimeout(() => {
+                this.smart_chart.setIsChartLoading(false);
+            }, 1500);
```

Comment:
I'm not sure about this 😅ideally we should fix the moving chart line first.

Reply:
Agreed as discussed, I don't believe it to be an ideal fix as well and it's not meant to be a fix but a temporary workaround. We will remove this once the issue with the moving line spots have been fixed on the chart. 🙏